apps/examples/board_specific_demo/ameba_secure_world_demo/secure_world.c

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************
+ *
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <security/security_common.h>
+#include <security/security_api.h>
+#include <tinyara/seclink.h>
+#include <tinyara/security_hal.h>
+#include <security/security_ss.h>
+#define SHOW_USAGE printf("Usage: secure_world -r -w\n");
+
+#ifdef  CONFIG_AMEBALITE_TRUSTZONE
+#define ADDR ((volatile uint32_t*)0x735C021)
```

Comment:
How about getting the addr with -a? If no input for addr, use this value as default. And help need to show valid range.